### PR TITLE
Use query-filters for CodeQL

### PR DESCRIPTION
**Pull request by sungam3r**
https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/customizing-code-scanning#excluding-specific-queries-from-analysis

### CODE REVIEW

File: .github/codeql-config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+query-filters:
+  - exclude:
+    id:
+      - cs/linq/missed-where
```
Comment (aeisenberg) [inline .github/codeql-config.yml:4]:
```suggestion
  - exclude:
         id:
             - cs/linq/missed-where
```

**Review by aeisenberg — COMMENTED** [review]
The `id` property is a sub-property of `exclude`.

File: .github/codeql-config.yml (hunk 1 above)
Comment (sungam3r) [reply .github/codeql-config.yml]: Thanks

**Comment by sungam3r** [comment]
11 warnings -> 3 warnings

**Pull request merged**